ocp_resources/daemonset.py

```diff
@@ -31,7 +31,7 @@ def wait_until_deployed(self, timeout=TIMEOUT):
         samples = TimeoutSampler(
             wait_timeout=timeout,
             sleep=1,
-            exceptions=ProtocolError,
+            exceptions_dict={ProtocolError: []},
```

Comment:
please save it under constant.py
PROTOCOL_ERROR_EXECEPTION_DICY = {ProtocolError: []}
and reuse it

Reply:
done.